pkgs/development/python-modules/crate/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchPypi
+, buildPythonPackage
+, urllib3
+, isPy3k
+, mock
+, sqlalchemy
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "crate";
+  version = "0.24.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "028q9r1qyqkq185awkazhplfy7y1081963fnjzi3kf3lxvz6yhay";
+  };
+
+  propagatedBuildInputs = [
+    urllib3
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    sqlalchemy
+    mock
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = "http://www.sqlalchemy.org/";
+    description = "A Python SQL toolkit and Object Relational Mapper";
+    license = licenses.mit;
```

Comment:
I would really like to have a maintainer of this

Reply:
Sure, myself. I also updated the whole meta attribute as this one was of a copy paste.